ios/RNSModalScreen.h

```diff
@@ -0,0 +1,21 @@
+#import "RNSScreen.h"
+
+@interface RCTConvert (RNSModalScreen)
+
++ (RNSScreenStackPresentation)RNSScreenStackPresentation:(id)json;
++ (RNSScreenStackAnimation)RNSScreenStackAnimation:(id)json;
+
+#if !TARGET_OS_TV
++ (RNSStatusBarStyle)RNSStatusBarStyle:(id)json;
++ (UIStatusBarAnimation)UIStatusBarAnimation:(id)json;
++ (UIInterfaceOrientationMask)UIInterfaceOrientationMask:(id)json;
```

Comment:
Why do we need to have statusbar and orientation oriented methods in RNSModalScreen?

Reply:
They come with `RCTConvert`. We could try and see if those are really needed in a follow up PR.